srcjs/notifications.js

```diff
@@ -0,0 +1,160 @@
+exports.notifications = (function() {
+
+  // Milliseconds to fade in or out
+  const fadeDuration = 250;
+
+  function show({ html=null, duration=5000, id=null, closeButton=true,
+                  style = null } = {})
```

Comment:
Have we called this kind of option (partial CSS class name) `style` in the past? To me it feels like an option called `style` should let you stuff CSS properties in there. `show({style: "font-weight: bold;"})`